### PR TITLE
Feat/Gokart

### PR DESCRIPTION
**Pull request by Zane-Larking**
Added CAN messages for the gokart.

Unresolved issues:
Written in the can.proto file are the bit lengths needed on the CAN frame for each value. **These types are not supported by protobuf**. The values are needed to be re-declared as int32/uint32, and within a ROS node or on the MoTec either needs values to be shift encoded into a single/two 32 bit integers or masking on each value values should be specified.

### CODE REVIEW

File: definitions.yml [hunk 1]
```diff
@@ -11,3 +11,32 @@ messages:
...
+
+messages:
+  - ID: 0x501
+    can_line: "can3"
+    message: "DrivingDynamics2"
+    
+messages:
+  - ID: 0x502
+    can_line: "can3"
+    message: "SystemStatus"
```
Comment (Jack17432) [inline definitions.yml]: this is a yml list so remove the messages: so it looks something like 

```
messages:
  - ID: 0x500
    can_line: "can3"
    message: "DrivingDynamics1"
  - ID: 0x501
    can_line: "can3"
    message: "DrivingDynamics2"
```

File: fs-common-protobuf/can.proto [hunk 2]
```diff
@@ -11,3 +11,102 @@ message AMSError {
...
+// value. Either values need to be shift encoded into a single/two 32 bit ints	
+// or masking on values should be specified.	
+
+/* Orin -> Nano */	
+/* Current ID: 0x300 */	
+/* size 6 bytes */	
+message Ackermann {	
+    int8 speed = 1; /* Desired speed of the car. Must be between 0 and 120 km/h. */	
+    int8 acceleration = 2; /* Desired acceleration. Must be between 0 and 256 m/s^2. */	
+    int8 jerk = 3; /* Desired jefk. Must be between 0 and 1 m/s^3. */
```
Comment (Jack17432) [inline fs-common-protobuf/can.proto]: spelling

**Review by Jack17432 — CHANGES_REQUESTED** [review]

File: definitions.yml (hunk 1 above)
Comment (Zane-Larking) [reply definitions.yml]: done

File: fs-common-protobuf/can.proto (hunk 2 above)
Comment (Zane-Larking) [reply fs-common-protobuf/can.proto]: jerf is now jerk.

**Review by Jack17432 — APPROVED** [review]
lgtm!

**Pull request merged**